internal/k8s/controller.go

```diff
@@ -342,6 +342,72 @@ func AddRouteEventHandler(numWorkers uint32, c *AviController) cache.ResourceEve
 	return routeEventHandler
 }
 
+func AddPodEventHandler(numWorkers uint32, c *AviController) cache.ResourceEventHandler {
+	podEventHandler := cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			if c.DisableSync {
+				return
+			}
+			if lib.IsNodePortMode() {
```

Comment:
Is this required? Since you are only calling this method on serviceType equals `NodePortLocal'?

Reply:
Removed.